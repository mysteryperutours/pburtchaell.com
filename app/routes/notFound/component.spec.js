import React from 'react';
import NotFoundRoute from './component';

describe(NotFoundRoute.name, () => {
  it('should render component', () => {
    expect(shallow(<NotFoundRoute />)).toMatchSnapshot();
  });
});
