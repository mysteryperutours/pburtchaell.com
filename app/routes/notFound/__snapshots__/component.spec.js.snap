exports[`NotFoundRoute should render component 1`] = `
<RouteContainer
  color={
    Object {
      "b": 255,
      "g": 255,
      "r": 255,
    }
  }
  defaultTheme={true}
  footer={true}
  header={true}>
  <Row
    defaultColumn={true}
    elementType="div"
    size="default">
    <Text
      type="h1">
      Not Found
    </Text>
    <hr />
    <Text
      type="p">
      Sorry, this page does not exist.
    </Text>
  </Row>
</RouteContainer>
`;
